vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/EmailField.java

```diff
@@ -37,13 +37,55 @@
 
 /**
  * Email Field is an extension of Text Field that only accepts email addresses
- * as input. If the given address is invalid, the field is highlighted in red
- * and an error message appears underneath the input. The validity of the email
- * addresses is checked according to the RFC 5322 standard, which includes the
- * format for email addresses. The component also supports supplying additional
- * validation criteria using a regular expression (see
- * {@link #setPattern(String)}). These extra validation criteria can be used,
+ * as input.
+ *
+ * <h2>Validation</h2>
+ * <p>
+ * Email Field comes with a built-in validation mechanism based on constraints.
+ * Validation is triggered when the user applies an email address by pressing
+ * Enter, blurring the field, etc, or when the value is updated
```

Comment:
Yeah, awkward, I agree. However, there are indeed more options, for example, clicking on the clear button. Could you recommend a way to put this to avoid giving the impression that the mentioned triggers are the only possible ones?

Reply:
Meanwhile, updated it to use "for example". Would it work better?

Reply:
We've agreed on https://github.com/vaadin/flow-components/pull/6544#discussion_r1730767594